### DISCUSSION TITLE
Move command option to retain source file timestamp

### DISCUSSION THREAD

**Discussion by hastyeagle** [beetbox/beets#5078] [Ideas]
First off -- awesome app.  I just found out about it last week and am loving the functionality of beets.  Great job!

One option I'd like to see added to the **move** command is for it to retain the source file's timestamp.  My use case is I move (copy) files to a directory that I then rsync over to an external drive for use in my car.  Currently, using "beet move -c -e -d ~/dest <filter>" copies the files as expected, but each file has a timestamp of when the file was copied, not of the original file.  When I then go to rsync that directory over to the external drive, it sees that every file has changed and copies them all over again.  It's not a huge problem, it just would speed things up slightly by keeping the original timestamp.

Thanks again!

**Comment by arogl** [comment]
In your `config.yaml` file add the following code snippet:

```yaml
plugins:
  importadded
 
importadded:
  preserve_mtimes: yes
```

Also great documentation located here [beets: the music geek’s media organizer docs](https://beets.readthedocs.io/en/stable/).

Feel free to suggest changes via PRs to update the documentation

**Reply by hastyeagle** [reply]
I already had that in my config when I ran the move / copy command, and the mtimes weren't preserved.  I'm on v1.6.0.

**Comment by arogl** [comment]
Have you tried running from git source?

Maybe this is a bug?

**Comment by mikestir** [comment]
I ran into this. `preserve_mtimes` only seems to work for the import command, but not for move. I would agree it would be nice if move/copy could do this as well.
